docs/howtos/squeeze_and_excite.rst

```diff
@@ -0,0 +1,32 @@
+Squeeze and Excite Block
+=============================
+
+We will show you how to...
+
+* Implement a Squeeze and Excite Block
+Squeeze and Excite Blocks were introduced in `"Squeeze-and-Excitation Networks" <https://arxiv.org/abs/1709.01507>`_ seek to
+improve the quality of representations learned by a network by modelling the dependencies between channels of convolutional features.
+The first step in the block is the Squeeze operation: all channels are aggregated by taking their means, changing the 
+tensor's shape from (H, W, C) to (1, 1, C). Then it goes through an Excitation operation: the mean vector passes through a 2 layer MLP
+and then is used in a self-gating mechanism, where it modulates each channel. The output still has shape (H, W, C) and so can just be passed
```

Comment:
Suggestion:

To avoid the "then... then" repetition, how about "then... before being..."

```suggestion
before being  used in a self-gating mechanism, where it modulates each channel. The output still has shape (H, W, C) and so can just be passed
```